### PR TITLE
bump somalier/extract and somalier/relate version

### PR DESCRIPTION
**Pull request by fellen31**
Version bump and nf-test migration of somalier extract and relate. 

Leaving somalier ancestry, since the latest version seems broken when running with containers, and I believe the module is not that used.

## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

File: modules/nf-core/somalier/relate/tests/main.nf.test.snap [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+                ]
+            }
+        ],
+        "meta": {
+            "nf-test": "0.8.4",
+            "nextflow": "24.04.2"
+        },
+        "timestamp": "2024-06-28T10:28:03.681901901"
+    },
+    "[ delete_me, []\u00a0], []": {
```
Comment (SPPearce) [inline modules/nf-core/somalier/relate/tests/main.nf.test.snap:73]: This is weird, unicode space? (and several more)

**Review by SPPearce — APPROVED** [review]
Some weird unicode spaces in the names of the tests in the nf-test snap, but other than that all fine.

**Comment by SPPearce** [comment]
I don't know why the tests aren't running, with a message about miniconda

File: modules/nf-core/somalier/relate/tests/main.nf.test.snap (hunk 1 above)
Comment (fellen31) [reply modules/nf-core/somalier/relate/tests/main.nf.test.snap:73]: Something is up with my terminal setup, will fix these.

**Comment by fellen31** [comment]
> I don't know why the tests aren't running, with a message about miniconda

Seems to be the same problem in other PRs.

**Review by SPPearce — APPROVED** [review]
Looks good to me.

**Pull request merged**